### PR TITLE
Add piq

### PR DESCRIPTION
**Pull request by jeongseok-meta**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jeongseok-meta** [comment]
@conda-forge/help-python, ready for review!

File: recipes/piq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/photosynthesis-team/piq/archive/v{{ version }}.tar.gz
+  sha256: beab5678dc0085cd473a23c57d3503aac3f985b344cc8ec7c21d62beda5ee446
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/piq/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

File: recipes/piq/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/piq/meta.yaml:15]: It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Comment (jeongseok-meta) [reply recipes/piq/meta.yaml:16]: Good to know. Changed the recipe back to a noarch package, skipping Windows

Comment (ocefpaf) [reply recipes/piq/meta.yaml:16]: You have to remove that skip, with it, it is not noarch.

File: recipes/piq/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{% set name = "piq" %}
```
Comment (ocefpaf) [inline recipes/piq/meta.yaml:1]:
```suggestion
{% set python_min = python_min|default("3.8") %}
{% set name = "piq" %}
```

**Comment by jeongseok-meta** [comment]
@ocefpaf thank you for the suggestions and merging this!

**Pull request merged**